### PR TITLE
Fixes - #9263 - Changed the symbol to use :)

### LINKED ISSUE #9263: Support for universal emojis
### Issue Summary

The Slack app in Ghost used to send notifications to Slack is now possible to use to send to Discord and other services supporting slack webhooks. This is great! But, I noticed that when I sent a test notification to Discord, the emoji `:simple-smile:` was not shown. This is because Discord has it as `:smile:`. This can easily be fixed if you use the universal form of the emoji, in this case the notification could send `:)` and Discord would translate it to `:smile:` while Slack would do its thing and translate to `:simple-smile:`. Also any chance for supporting emojis in the stories?

### Steps to Reproduce

1. Add a Discord webhook (followed by the postfix /slack) into Ghost.
2. Send a test notification.
3. Notice that the emoji is not shown because the emoji name is wrong.

### Technical details:

* Ghost Version: v1.17.3
* Node Version: 8.9.0
* Browser/OS: Safari on macOS High Sierra
* Database: SQLite3

### PR DESCRIPTION
**Pull request by Abijeet**
Fixes #9263 

- [x] There's a clear use-case for this code change
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `npm test`)

### CODE REVIEW

File: core/server/services/slack.js [hunk 1]
```diff
@@ -98,7 +98,7 @@ function listener(model, options) {
 
 function testPing() {
     ping({
-        message: 'Heya! This is a test notification from your Ghost blog :simple_smile:. Seems to work fine!'
+        message: 'Heya! This is a test notification from your Ghost blog :). Seems to work fine!'
```
Comment (AileenCGN) [inline core/server/services/slack.js]: Tested with `:)` and this doesn't get rendered as emoji in Slack. I suggest, we change it to `:smile:` which - apparently - will work for both, Slack (I tested it and it works) and Discord.

**Review by aileen — CHANGES_REQUESTED** [review]
Hey @Abijeet 

Thanks for the PR. I don't know, if you've tested this with a Slack webhook, but changing the `:simple-smile:` to `:)` didn't seem to work.

Maybe we just rename it to `:smile:`, which seems to work fine with Slack.

File: core/server/services/slack.js (hunk 1 above)
Comment (AileenCGN) [reply core/server/services/slack.js]: image

See here ^ first message with `:)` and second with `:smile:`.

**Comment by Abijeet** [comment]
@AileenCGN - Thanks for doing the bulk of the work for me. That was sloppy. I tried to test it, but got an error because I was copying a space from my slack web-hook URL to the slack integration URL in the admin settings. This was causing Ghost to complain about invalid URL. 

I was initially unable to understand why this was happening, but figured out when I went back this time.

I've now gone ahead and opened a PR to **fix** the above as well - https://github.com/TryGhost/Ghost-Admin/pull/925 . Please merge it if you think it's needed.

Adding an image this time,

9263-test-case

**Review by aileen — APPROVED** [review]

**Pull request merged**